src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,51 +107,107 @@ const CrossPodWriteForm = () => {
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="upload-doc-expiration"
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expire date (if applicable)"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+                slotProps={{
+                  textField: {
+                    helperText: 'MM/DD/YYYY'
+                  }
+                }}
+              />
+            </LocalizationProvider>
+          </FormControl>
           <br />
-          <textarea id="upload-doc-desc" name="description" {...description} />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doctype">File to upload: </label>
-          <input
-            id="upload-doctype"
-            type="file"
-            name="uploadDoctype"
-            accept=".pdf, .docx, .doc, .txt, .rtf, .gif"
-            onChange={handleFileChange}
-          />
-          <button disabled={state.processing} type="submit">
-            Upload file
-          </button>
-        </div>
-      </form>
+          <br />
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* <Typography htmlFor="upload-doc-desc">Enter description:</Typography> */}
+            <TextField
+              id="upload-doc-desc"
+              name="description"
+              multiline
+              rows={4}
+              label="Enter description"
+              value={docDescription}
+              onChange={(newDocDescription) => setDocDescription(newDocDescription.target.value)}
+              placeholder="Add a description here"
+            />
+          </FormControl>
+          <br />
+          <br />
+          <FormControl fullWidth>
+            <Button
+              variant="contained"
+              component="label"
+              color="primary"
+              id="upload-doctype"
+              name="uploadDoctype"
+              onChange={handleFileChange}
+              required
+            >
+              Choose file
+              <input type="file" hidden accept=".pdf, .docx, .doc, .txt, .rtf" multiple />
+            </Button>
+            <FormHelperText
+              sx={{
+                display: 'inline-block',
```

Comment:
Removing this attribute and the !important in Line 188 doesn't seem to affect the styling of helper text.

![Screen Shot 2023-06-14 at 15 04 17 (2)](https://github.com/codeforpdx/PASS/assets/14917816/dc33bb6b-55ea-4139-a8db-61674182a606)

Reply:
It doesn't seem like !important in Line 188 affects the styling of the helper Text.

Without `!important`:

![Screen Shot 2023-06-15 at 17 18 06 (2)](https://github.com/codeforpdx/PASS/assets/14917816/cb2bd322-cf9d-45ae-98af-24670497d8c7)

With `!important`:

![Screen Shot 2023-06-15 at 17 18 19 (2)](https://github.com/codeforpdx/PASS/assets/14917816/316c5c92-0c24-4f42-b7ef-29e58ef01316)

It's probably because we don't have to force the styling. It just conforms to MUI as expected.